packages/modul-components/src/utils/form/form-group.ts

```diff
@@ -141,12 +145,20 @@ export class FormGroup<T = any> extends AbstractControl {
         Vue.prototype.$set(this._controls, name, control);
     }
 
-    public removeControl(name: string): void {
-        if (this._controls[name] === undefined) {
-            throw Error(`There is no control with name ${name} in this group`);
+    public setControl(name: string, control: AbstractControl): void {
+        if (this._controls[name] !== undefined) {
```

Comment:
Je crois que ce if n'est pas nécessaire.

Reply:
Est-ce que le $delete fait erreur si ça n'existe pas? @guignol1981

Reply:
Non, enlève le bloc if au complet, y compris le $delete. Je ne crois pas qu'on ait besoin de faire un $delete avant de faire un $set.

Reply:
ça prend le delete: [set doc](https://vuejs.org/v2/api/#Vue-set). La doc parle d'ajouter dans ce cas ci et non de ajouter/remplacer.

Ceci dit, au lieu du if j'utiliserais this.removeControl(name);

Reply:
Je crois que le delete n'est pas nécessaire et que la documentation est imprécise :-P Va falloir que @setur52 le teste :-)

Reply:
peut-être une affaire de doc en effet!

Reply:
Selon les tests unitaire ça fonctionne :)